examples/cms-sanity/lib/sanity.js

```diff
@@ -0,0 +1,14 @@
+import sanityClient from '@sanity/client'
+
+const options = {
+  // Find your project ID and dataset in `sanity.json` in your studio project
+  projectId: 'lsd8pmjw',
```

Comment:
Are we ok hardcoding this, or should we use the env var?

Reply:
Moved it to `env` 👍